BootloaderCommonPkg/Include/Library/FitLib.h

```diff
@@ -0,0 +1,62 @@
+/** @file
+  FIT Load Image Support
+Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
```

Comment:
should we change the header to 2024?

Reply:
This file comes from https://github.com/tianocore/edk2/blob/master/UefiPayloadPkg/PayloadLoaderPeim/FitLib.h with minor update. Changing copy right year is optional.